backends/src/lib.rs

```diff
@@ -41,7 +41,7 @@ fn powers_of_two(max_value: usize) -> Vec<usize> {
 
 fn is_hpu() -> bool {
     match Command::new("hl-smi")
-        .args(&["-Q", "name", "-f", "csv"])
+        .args(["-Q", "name", "-f", "csv"])
```

Comment:
It's better to add `&`, but it is also OK to delete it.

Reply:
Clippy suggests not adding unnecessary references, this was part of its warnings.